src/services/codefixes/fixes.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference path="correctQualifiedNameToIndexedAccessType.ts" />
 /// <reference path="fixClassIncorrectlyImplementsInterface.ts" />

```

Comment:
Looks like there is an analogous entry to this in `src/harness/tsconfig.json`. Not sure why it's there, or if we might want to remove it. Or alternatively, add a reference to `correctQualifiedNameToIndexedAccessType.ts`.

@weswigham do you know if building for tests will correctly trigger a rebuild when we make changes to `correctQualifiedNameToIndexedAccessType.ts`?